extra_tests/snippets/builtin_range.py

```
@@ -118,6 +118,15 @@
 assert range(10, 1, -2).__reduce__()[0] == range
 assert range(10, 1, -2).__reduce__()[1] == (10, 1, -2)
 
+# range iterator __reduce__ (state is None, range rebased to current position)
+it = iter(range(10))
+next(it); next(it); next(it)
```

Flake8 (7.3.0)

[error] multiple statements on one line (semicolon)

(E702)